File: Java/Card.java
import java.util.Map;

class Card extends Payment{
    Integer number;
    Integer cvv;
    Map<String, Map<String, Integer>> date;

    public Card(Integer id, Integer number, Integer cvv, Map<String, Map<String, Integer>> date){
        super(id);
        this.number = number;
        this.cvv = cvv;
        this.date = date;
    }

}

File: PHP/Card.php
<?php
require_once("Payment.php")
class Card extends Payment {
    public $number;
    public $cvv;
    public $date;

    public function __construct($id, $number, $cvv, $date){
        parent::__construct($id);
        $this->number = $number;
        $this->cvv = $cvv;
        $this->date = $date;
    }
}
?>